.travis.yml

```diff
@@ -1,26 +1,42 @@
 language: generic
+dist: trusty
```

Comment:
I'm not sure if the OS version makes any difference in this case. I see no reason to lock it, if it doesn't matter.